packages/rpc-core/src/rpc-methods/common.ts

```diff
@@ -29,6 +29,15 @@ export type Slot = U64UnsafeBeyond2Pow53Minus1;
 // the JSON-RPC transport.
 export type U64UnsafeBeyond2Pow53Minus1 = bigint;
 
+// FIXME(solana-labs/solana/issues/30341)
+// <https://stackoverflow.com/questions/45929493/node-js-maximum-safe-floating-point-number/57225494#57225494>
+// Beware that floating-point value precision can vary widely:
+// - For precision of 1 decimal place, anything above 562949953421311
+// - For precision of 2 decimal places, anything above 70368744177663
+// can be truncated or rounded because of a downcast to JavaScript `number` between your calling
+// code and the JSON-RPC transport.
+export type F64UnsafeSingleDecimalPointBeyond2Pow53Over16Minus1 = number;
```

Comment:
I mean, I love it, but probably better to just be like `F64UnsafeSeeCodeCommentForWhy` or something :)

Reply:
I figured the longer the name the more obvious of a warning it is. Your suggestion is also quite long

Reply:
`__F64_UNSAFE___DO_NOT_USE_OR_YOU_WILL_BE_FIRED`

Reply:
I went with `F64UnsafeSeeDocumentation`